app/src/main/java/net/squanchy/eventdetails/domain/view/ExperienceLevel.java

```diff
@@ -36,4 +28,9 @@ public int rawLevel() {
     public int labelStringResId() {
         return labelStringResId;
     }
+
+    @IntRange(from = 0, to = 2)
+    public static int getRawLevel(String level) {
```

Comment:
This is not needed if you use the enum (you can `switch` in the View, the enum doesn't need to know about this)